### ISSUE TITLE
Customize Font + Size in Response

### ISSUE THREAD

**Issue by NghiaTranUIT** [ProxymanApp/Proxyman#431]
### 👏 Description
It would be great if the font and size are customizable from Preference app.

### Acceptance Criteria
- [ ] Add UI to Preference to select Font + Size
- [ ] Able to change font + size in Request/Response Panel (Raw Preview, JSON, Previewer, ...)

**Comment by donileo** [comment]
Thanks @NghiaTranUIT. Will file future issues here.

**Comment by NghiaTranUIT** [comment]
Bump to Proxyman 1.22.0 👍

**Comment by NghiaTranUIT** [comment]
@donileo let checkout this BETA: https://proxyman.s3.us-east-2.amazonaws.com/beta/Proxyman_1.21.0_Font_Size.dmg

We're able to customize the font size to fit your monitor and easier for reading 🌮 

<img width="702" alt="Screen Shot 2020-04-23 at 23 17 42" src="https://user-images.githubusercontent.com/5878421/80123334-f0e15f80-85b8-11ea-99d1-8eb37c61597f.png">

<img width="2416" alt="Screen Shot 2020-04-23 at 23 20 58" src="https://user-images.githubusercontent.com/5878421/80123440-1a01f000-85b9-11ea-9ae0-45a5a7a0368e.png">

**Comment by donileo** [comment]
Hi, thanks for the effort here. Is there a way to change the font? My original intention was just that, to allow picking a specific font and size to use for showing the responses headers etc.

**Comment by NghiaTranUIT** [comment]
Since the Request/Response Body Previewer is built on top of [CodeMirror](https://codemirror.net), so sometime it won't work properly with system fonts. I attempted but no luck : /

Thus, I don't support customize font at this moment. I will research it further and support it soon.

**Comment by donileo** [comment]
Excellent!

**Comment by NghiaTranUIT** [comment]
Do you mean the font? @ghilyusha 

What is the current font issue that you'd like to change 🤔 Are you prefer using the monospace font?

**Comment by NghiaTranUIT** [comment]
@ghilyusha yes, Monaco Editor is a new replacement for CodeMirror. I still don't understand why do you need to customize the Font Name? Is the current font hard to see? 🤔

**Comment by ghilyusha** [comment]
It's quite standard for applications to allow a custom monospace font family to accommodate for the preference of the user. I can find a setting for this in the majority of dev tools, such as VSCode, Paw/RapidAPI, Postman, Fork, iTerm, Xcode, etc.

The use case for this is allow the user to have the same monospace font everywhere.

Furthermore, the current font used by Monaco (called Menlo) is deprecated on macOS in favor of a new one (called SF Mono).

**Comment by NghiaTranUIT** [comment]
Thanks for the clarification. If I add the "Use monospace Font", does it fulfill your need? 

Even though I support Custom Font or Monospace Font, the new font only applies to the Body, Header Key/Value, and Content, ... not every font of the app is changed because it's really hard to customize every single UI Control of the app.

**Comment by ghilyusha** [comment]
I'm sorry for the confusion. I only meant to ask for a setting that changes the font for code, such as the request previews, map local tool, etc. That means only the bits that are rendered in Monaco.

**Comment by NghiaTranUIT** [comment]
Ah thanks, I understand now 👍 Will checkout how much workload to support Font for the Monaco Editor

**Comment by isadon** [comment]
Any update on this? Been a long time.

**Comment by NghiaTranUIT** [comment]
It's already been in the latest app. You can change the font size or use Monosapced font of the Monaco Editor (Body, Map Local and Scripting)
 
<img width="1840" height="1117" alt="Image" src="https://github.com/user-attachments/assets/56111447-d5c8-4998-a1c4-266fc02a72e2" />

We don't allow to change the font, because I don't think it's useful,
